.github/workflows/build-gh-pages.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+name: Build website and publish to GH pages
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Try enabling yamllint in your editor to catch this inconsistent indent.

Reply:
Thanks, done